### PR TITLE
Fix/linkifyscreen

### LINKED ISSUE #42: Link instead of full URL for the screenshot?
image

### PR DESCRIPTION
**Pull request by tbushman**
Closes #42

### CODE REVIEW

**Comment by tbushman** [comment]
This wraps the screenshot in an anchor with href for the image url.

**Comment by scissorsneedfoodtoo** [comment]
Everything looks good, just noticed a missing comma. I'll leave a comment on the line

File: server/views/layout.pug [hunk 1]
```diff
@@ -33,7 +33,7 @@ html(lang='en')
               collaborators: 'Collaborators',
               screenshot: 'Screenshot'
             },
-            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance'],
+            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance']
```
Comment (scissorsneedfoodtoo) [inline server/views/layout.pug]: I think there's just a missing comma here. Once I added one locally everything was working great!

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
I think once that's added this will be good to go!

File: server/views/layout.pug (hunk 1 above)
Comment (tbushman) [reply server/views/layout.pug]: Thanks. I had a merge conflict debacle today. Thank you.

**Pull request merged**